packages/shared/src/services/etherscan/EtherscanClient.ts

```diff
@@ -1,83 +1,56 @@
-import { EthereumAddress, Hash256, RateLimiter } from '@l2beat/shared-pure'
-
+import {
+  ChainId,
+  EthereumAddress,
+  Hash256,
+  UnixTime,
+} from '@l2beat/shared-pure'
+
+import { Logger } from '../../tools'
+import { stringAsInt } from '../../tools/types'
+import { EtherscanLikeClient } from '../etherscanlike/EtherscanLikeClient'
 import { HttpClient } from '../HttpClient'
+import { BlockNumberProvider } from '../providers/BlockNumberProvider'
 import {
   ContractCreatorAndCreationTxHashResult,
   ContractSourceResult,
-  EtherscanResponse,
 } from './model'
 
-export class EtherscanClient {
-  private readonly rateLimiter = new RateLimiter({
-    callsPerMinute: 120,
-  })
-  private readonly timeoutMs = 10_000
+export class EtherscanError extends Error {}
+
+export class EtherscanClient
```

Comment:
There are two EtherscanClients, this one is used only in tests, what is the purpose of this?

Reply:
I forgot to remove the one from backend, my fault